### PR TITLE
Fix typos across repo

### PR DESCRIPTION
**Pull request by mgaeta**
I ran my IDE's spellchecker on the repo. I've separated different classes of changes into commits. This is not yet a comprehensive fix for all typos, more probably remain.

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 def pytest_configure(config):
     import warnings
 
-    # XXX(dramer): Kombu throws a warning due to transaction.commit_manually
+    # XXX(dcramer): Kombu throws a warning due to transaction.commit_manually
```
Comment (dashed) [inline conftest.py:18]: 🤣

**Comment by dashed** [comment]
@mgaeta which IDE?

**Comment by mgaeta** [comment]
> @mgaeta which IDE?

PyCharm

**Review by manuzope — COMMENTED** [review]
All the changes bar those two comments LGTM.

File: src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/index.tsx [hunk 2]
```diff
@@ -71,7 +71,7 @@ const POLLING_MAX_TIME_LIMIT = 3 * 60000;
 type ConditionOrAction = string;
 
 type RuleTaskResponse = {
-  status: 'pending' | 'failed' | 'succes';
+  status: 'pending' | 'failed' | 'success';
```
[the remark this answers is not in the document]
Comment (dashed) [reply src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/index.tsx:74]: @billyvg can you comment on this?

[the remark this answers is not in the document]
Comment (billyvg) [reply src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/index.tsx:74]: Looks green to me

File: src/sentry/static/sentry/app/utils/eventWaiter.tsx [hunk 3]
```diff
@@ -9,14 +9,14 @@ import withApi from 'app/utils/withApi';
 const DEFAULT_POLL_INTERVAL = 5000;
 
 const recordAnalyticsFirstEvent = ({organization, project}) =>
-  analytics('onboarding_v2.first_event_recieved', {
+  analytics('onboarding_v2.first_event_received', {
```
[the remark this answers is not in the document]
Comment (mgaeta) [reply src/sentry/static/sentry/app/utils/eventWaiter.tsx:12]: I think @mitsuhiko is considering removing the log entirely?

**Comment by mgaeta** [comment]
Just rebased to resolve merge conflicts.

**Review by manuzope — APPROVED** [review]

**Pull request merged**